### DISCUSSION TITLE
how do i remove the vertical margin line on the right of the editor

### DISCUSSION THREAD

**Discussion by khisareuben** [pulsar-edit/.github#253] [Pulsar Support] [answered]
There is a vertical margin line at the right of my pulsar editor and i have looked for ways to remove it with no success. can someone help me
![pulsar](https://github.com/pulsar-edit/.github/assets/156296475/a63e94ae-12ee-4779-88bf-48956c641fbe)

**Comment by Spiker985** [comment, marked as answer]
Packages -> wrap-guide -> Settings -> Enabled

Alternatively, you can have it only display when you have soft-wrap or soft-wrap at preferred line length - are enabled

**Reply by khisareuben** [reply]
Thanks for that

**Reply by Spiker985** [reply]
> Thanks for that

@khisareuben No problem, if you can - please mark this as answered that way it suggests it to people when they go to write similar discussions and issues
